src/file-box.ts

```diff
@@ -1017,14 +1017,12 @@ class FileBox implements Pipeable, FileBoxInterface {
   pipe<T extends Writable> (
     destination: T,
   ): T {
-    this.toStream().then(stream => {
-      stream.on('error', e => {
-        console.info('error:', e)
-
-        destination.emit('error', e)
+    this.toStream()
+      .then(stream => {
+        stream.once('error', e => destination.destroy(e))
```

Comment:
Why do you destroy the destination instead of emitting `error` when there's an error?

Reply:
Yes, only emit error is needed here, not destroy error.